test/schema.sql

```diff
@@ -164,3 +164,14 @@ CREATE TABLE messages (
   INDEX type (type),
   INDEX receipt (receipt)
 );
+
+CREATE TABLE user_space_activities (
+  user_id varchar(64) NOT NULL,
+  space_id varchar(64) NOT NULL,
```

Comment:
SQL command should be uppercase, also we should use same naming than what we have on others tables 
```suggestion
  user VARCHAR(64) NOT NULL,
  space VARCHAR(64) NOT NULL,
```

Reply:
fixed